aspnetcore/signalr/messagepackhubprotocol.md

```diff
@@ -172,6 +192,30 @@ InvalidDataException: Error binding arguments. Make sure that the types of the p
 
 For more information on this limitation, see GitHub issue [aspnet/SignalR#2937](https://github.com/aspnet/SignalR/issues/2937).
 
+### Passing Class information in Java
```

Comment:
This isn't specific to MessagePack is it?

Reply:
No, but it's only required in MessagePack, JSON still works without it. I could move this to java-client.md and call out the differences there.

Reply:
How does JSON manage to work without it given the type erasure? It's also probably worth pointing out type erasure is the reason a TypeReference is necessary.

Reply:
Oh, you're right, I was remembering wrong - JSON had the same issue with more complex types but we hadn't noticed before. I'll move this segment & expand it now.

Reply:
https://github.com/dotnet/AspNetCore.Docs/pull/21090/commits/fd51aba3760da7845281f792159238d121ee3741